sjsonnet/src/sjsonnet/Importer.scala

```diff
@@ -158,7 +158,7 @@ case class StaticResolvedFile(content: String) extends ResolvedFile {
   def readString(): String = content
 
   // We just cheat, the content hash can be the content itself for static imports
-  lazy val contentHash: String = content
+  def contentHash(): String = content
```

Comment:
question: Here is another swap from `lazy val` to `def` as @lomigmegard pointed. What was the reasoning to change it?